shell/protoc.sh

```diff
@@ -211,6 +211,16 @@ if has_grpc_client "node"; then
   )
 
   "$grpc_tools_node_bin" "${ts_args[@]}" "$(get_repo_directory)$workDir$SUBDIR/"*.proto
+
+  # remove unsupported validate proto plugin
+  if pushd "$grpc_path"; then
+    for f in * **/*; do
```

Comment:
Isn't this the same as `* */*` in bash?

Any reason not to use a `find` command, like `for f in $(find . -name '*.js')` or similar?

Reply:
Not quite. `* **/*` will recursively go as deep as it needs to in the directory tree. `* */*` will only go one level deep.

Generally it's considered bad taste to use `find` for things that glob can handle. Plus everything in this directory should only be auto generated stuff.

Reply:
This isn't true with the version on macos (5.2):

```bash
bash-5.2$ mkdir -p this/is/a/long/path
bash-5.2$ touch this/is/a/long/path/example.txt
bash-5.2$ cd this/
bash-5.2$ ls **/*
long
bash-5.2$ ls * **/*
is:
a

is/a:
long
bash-5.2$ 
```

Reply:
Oh yeah, I totally forgot about `globstar` being off by default on Macs and older bash versions. This new pattern should handle all the cases.